strongbox-web-core/src/main/resources/application.properties

```diff
@@ -16,3 +20,5 @@ logging.dir=strongbox-vault/logs
 logging.config.file=strongbox/etc/logback-debug.xml
 java.io.tmpdir=strongbox-vault/tmp
 ehcache.disk.store.dir=strongbox/cache
+
+spring.main.allow-bean-definition-overriding=true
```

Comment:
Without this, it throws:

```
***************************
APPLICATION FAILED TO START
***************************

Description:

The bean 'authenticationManager', defined in class path resource [org/carlspring/strongbox/configuration/StrongboxSecurityConfig.class], could not be registered. A bean with that name has already been defined in URL [jar:file:/home/carlspring/.m2/repository/org/carlspring/strongbox/strongbox-security-api/1.0-SNAPSHOT/strongbox-security-api-1.0-SNAPSHOT.jar!/org/carlspring/strongbox/security/managers/AuthenticationManager.class] and overriding is disabled.

Action:

Consider renaming one of the beans or enabling overriding by setting spring.main.allow-bean-definition-overriding=true
```

@fuss86 , @ptirador : Could you maybe have a look?

Reply:
Have you tried erasing 

```
@Component
```
from `org.carlspring.strongbox.security.managers.AuthenticationManager` ?

Reply:
@ptirador : No. Would you like to poke around?

Reply:
@carlspring: Solved. Everything seems working now...

Reply:
Thanks a lot! :)

Reply:
@ptirador : This seems to have broken the build...

Reply:
Hmm, it's weird, everything worked in my local environment...